ext/opentelemetry-ext-asgi/tests/test_asgi_middleware.py

```diff
@@ -0,0 +1,330 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import unittest
+import unittest.mock as mock
+
+import opentelemetry.ext.asgi as otel_asgi
+from opentelemetry import trace as trace_api
+from opentelemetry.test.asgitestutil import (
+    AsgiTestBase,
+    setup_testing_defaults,
+)
+
+
+async def http_app(scope, receive, send):
+    payload = await receive()
```

Comment:
I'd express the same idiom nit here as in the middleware implementation, i.e. `receive()` and `send()` pass around "messages" instead of "payloads".